compaction/compaction_manager.cc

```diff
@@ -1133,17 +1135,19 @@ class compaction_manager::rewrite_sstables_compaction_task : public compaction_m
 
 protected:
     virtual future<> do_run() override {
+        compaction_task_stats cts{};
         switch_state(state::pending);
         auto maintenance_permit = co_await acquire_semaphore(_cm._maintenance_ops_sem);
 
         while (!_sstables.empty() && can_proceed()) {
             auto sst = consume_sstable();
-            co_await rewrite_sstable(std::move(sst));
+            const auto& res = co_await rewrite_sstable(std::move(sst));
+            cts.validation_errors += res.validation_errors;
```

Comment:
Should FIXME comment be added here? Since currently `cts` is not used anywhere (it is in the next commit).

Reply:
yes, please